### PR TITLE
init with DB

### PR DESCRIPTION
**Pull request by Michalikhol**
initial project with connection to mongo

### CODE REVIEW

File: .env
Comment (shirshponder) [inline .env:1]: think we should not save env in git

File: controllers/posts_controller.js
Comment (shirshponder) [inline controllers/posts_controller.js:1]: i think we should decide on a pattern for the names of files and functions. i think it is better camelCase

File: controllers/posts_controller.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+const getPostById = async (req, res) => {
+  const postId = req.params.id;
+  console.log(postId)
+
+  try {
+    const post = await PostModel.findById(postId);
+    if (post) {
+      res.send(post);
+    } else {
+      res.status(404).send("Post not found");
```
Comment (shirshponder) [inline controllers/posts_controller.js]: maybe use a library for status code

File: models/posts_model.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    required: true,
+  },
+  content: String,
+  owner: {
+    type: String,
+    required: true,
+  },
+});
+
+const postModel = mongoose.model("Posts", postSchema);
```
Comment (shirshponder) [inline models/posts_model.js]: Should the collection start with a lowercase?

File: request.rest [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+###
+
+POST http://localhost:3000/posts
+Content-Type: application/json
+
+{
+    "title": "Hello Shir",
+    "owner": "Michali",
+    "content": "Shir is the best"
```
Comment (shirshponder) [inline request.rest]: michali is better

File: controllers/posts_controller.js
Comment (Michalikhol) [reply controllers/posts_controller.js:1]: I agree

**Review by shirshponder — APPROVED** [review]

**Pull request closed without merging**